bundles/CoreBundle/DependencyInjection/PimcoreCoreExtension.php

```diff
@@ -85,8 +85,6 @@ public function loadInternal(array $config, ContainerBuilder $container)
 
         $container->setParameter('pimcore.web_profiler.toolbar.excluded_routes', $config['web_profiler']['toolbar']['excluded_routes']);
 
-        $container->setParameter('pimcore.response_exception_listener.render_error_document', $config['error_handling']['render_error_document']);
```

Comment:
I'm afraid we need to keep this parameter as well -> BC break. 
But please add a deprecation note: `// @deprecated since Pimcore 10.1, parameter will be removed in Pimcore 11`.

Reply:
@brusch This is done (i changed by the way the node option deprecated message with same information you gave) !